JDBC/ConnectionUtil.java

```diff
@@ -0,0 +1,30 @@
+package com.metacube.jdbc.example;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class ConnectionUtil {
+
+	private static final String DB_URL = "jdbc:mysql://localhost:3306/library";
+	private static final String USER = "root";
+	private static final String PASSWORD = "mysql";		
+	private Connection con = null;
+	public Connection getConnection() {
+		/* register driver */
+		try {
+			Class.forName("com.mysql.jdbc.Driver");
+		} catch (ClassNotFoundException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+		/* open connection */
+		try {
+			con = DriverManager.getConnection(DB_URL, USER, PASSWORD);
```

Comment:
Instead of writing two try catch block, you can write both lines in one try block and that can be followed by two catch block.